### PR TITLE
check-before-deployment: add selinux

### PR DESCRIPTION
**Pull request by dveeden**
See also:
- https://docs.pingcap.com/tidb/stable/deploy-a-dm-cluster-using-tiup
- https://docs.pingcap.com/tidb/stable/tiup-component-cluster-check#selinux

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: check-before-deployment.md [hunk 1]
```diff
@@ -781,3 +781,8 @@ sudo yum -y install numactl
     ```
 
     To get help information of the `tiup cluster exec` command, run the `tiup cluster exec --help` command.
+
+## Disable SELinux
+
+Check with the [`getenforce(8)`](https://linux.die.net/man/8/getenforce) utility to see if SELinux is disabled or set to permissive. When SELinux is in Enforcing mode it can cause deployment failures. Refer to the documentation of your Operating System to see how to disable SELinux.
```
Comment (hfxsd) [inline check-before-deployment.md:787]:
```suggestion
Use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility to check if SELinux is disabled or set to permissive. SELinux in Enforcing mode can cause deployment failures. Consult your operating system's documentation for instructions on disabling SELinux.
```

File: dm/quick-start-with-dm.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs/tidb-data-migration/dev/get-started/']
 
 This document describes how to migrate data from MySQL to TiDB using [TiDB Data Migration (DM)](/dm/dm-overview.md). This guide is a quick demo of DM features and is not recommended for any production environment.
 
+> **Note:**
+>
+> If your target machine's operating system supports SELinux, make sure that SELinux is **disabled**.
```
Comment (hfxsd) [inline dm/quick-start-with-dm.md:13]:
```suggestion
> If your target machine's operating system supports SELinux, ensure that SELinux is **disabled**.
```

**Review by hfxsd — APPROVED** [review]

File: check-before-deployment.md [hunk 3]
```diff
@@ -781,3 +781,8 @@ sudo yum -y install numactl
...
+
```
Comment (hfxsd) [inline check-before-deployment.md:788]:
```suggestion
```

File: check-before-deployment.md [hunk 4]
```diff
@@ -781,3 +781,7 @@ sudo yum -y install numactl
     ```
 
     To get help information of the `tiup cluster exec` command, run the `tiup cluster exec --help` command.
+
+## Disable SELinux
+
+Use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility to check if SELinux is disabled or set to permissive. SELinux in Enforcing mode can cause deployment failures. Consult your operating system's documentation for instructions on disabling SELinux.
```
Comment (qiancai) [inline check-before-deployment.md:787]:
```suggestion
Use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility to check if SELinux is disabled or set to permissive. SELinux in enforcing mode can cause deployment failures. For instructions on disabling SELinux, refer to your operating system's documentation.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Comment by qiancai** [comment]
@dveeden Please involve a tech reviewer for this PR.

**Review by lance6716 — APPROVED** [review]

**Pull request merged**